src/adapters/postgres/rbac/privilege-adapter.ts

```
@@ -223,6 +225,18 @@ export class PostgresPrivilegeService {
           HttpStatus.NOT_FOUND
         );
       }
+      // Capture affected tenants before removing the mappings, so their
+      // cached effective privileges can be invalidated below.
+      const affectedMappings = await this.rolePrivilegeMappingRepository.find(
+        {
```

ESLint

[error] Delete `⏎········`

(prettier/prettier)